docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,703 @@
+. Once installed, you can reference a collection content by its fully qualified collection name (FQCN).
+
+NOTE: The `create_kafka` command in the task needs to be the FQCN of the module for collection development, for example
+`rhoas.rhoas.create_kafka`.
+
+
+[id="proc-using-rhoas-ansible-collection_{context}"]
+=== Using the RHOAS Ansible collection
+
+.Prerequisites
+* You have a Red Hat account.
+* You have an offline token that is accessible to Ansible.
+
+[NOTE]
+The offline token is a refresh token with no expiry and can be used by non-interactive processes to provide an access token for {product-long-rhoas} to authenticate the user. The token is an OpenShift offline token and can be found at https://cloud.redhat.com/openshift/token.
+
+* Two further environment variables are used for the collection to work. These environment variables can be placed in a `.env` file. If neither of these environment variables are set, the collection will default to the URLs shown in the examples.
+- API_BASE_HOST - The base host for the API. This is the base URL for the API. For example, https://api.openshift.com.
+- SSO_BASE_HOST - The base host for the SSO. This is the base URL for the SSO. For example, https://sso.redhat.com/auth/realms/redhat-external.
+
+.Procedure
+
+. Once the collection is installed, documentation for each module can be found by entering `ansible-doc rhoas.rhoas.<module_name>` and replacing <module_name> with the name of the module. The following example outputs the documentation for the `create_kafka` module. It shows the required (denoted by `=`) and optional parameters (denoted by `-`) for the module, along with examples of how to use the module, and the output of the module.
++
+[source,shell]
+----
+$ ansible-doc rhoas.rhoas.create_kafka
+----
+. You can run a single module to complete one task like in the following example which creates a Kafka instance.
++
+.Example create_kafka module
+[source,shell]
+----
+$ ansible localhost -m rhoas.rhoas.create_kafka -a 'name=unique-kafka-name billing_model=standard cloud_provider=aws plan="developer.x1" region="us-east-1" openshift_offline_token=<OFFLINE_TOKEN>'
+----
+. Alternatively, you can run a playbook. Enter the following command to run the https://github.com/redhat-developer/app-services-ansible/blob/main/rhoas_test.yml[rhoas test playbook].
++
+[source,shell]
+----
+$ ansible-playbook rhoas_test.yml
+----
+[NOTE]
+The openshift_offline_token is used for authentication to enable communication with the Kafka Management API. If the token is not passed in as an argument per task, the module attempts to read it from the `_OFFLINE_TOKEN_` environment variable.
+
++
+NOTE: The example playbook used in this section includes comments that indicate how to directly specify values rather than fetching them dynamically. For example, to specify a Kafka instance ID, a comment in the playbook states that you can include the following line:
++
+[source]
+----
+kafka_id: <kafka_id>
+----
```

Comment:
To italicize the replaceable value:

```
[source, subs="+quotes"]
----
kafka_id: __<kafka_id>__
----
```

Reply:
In the current HTML preview, you'll see that the `_kafka_id:` example is not enclosed within the note. To do this, you need to add a block-style note, as shown below:

```
[NOTE] 
====
The example playbook used in this section includes comments that indicate how to directly specify values rather than fetching them dynamically. For example, to specify a Kafka instance ID, a comment in the playbook states that you can include the following line:

[source, subs="+quotes"]
----
kafka_id: __<kafka_id>__
----
====
```